scripts/lib/config.sh

```diff
@@ -81,10 +81,10 @@ function create_gcloud_config {
   export CLOUDSDK_ACTIVE_CONFIG_NAME="${CONFIG_NAME}"
```

Comment:
this already sets the active config

Reply:
I'll debug today, but this wasn't working for me until I made this change.